packages/react/src/components/Rating/Rating.base.tsx

```diff
@@ -139,8 +141,10 @@ export const RatingBase: React.FunctionComponent<IRatingProps> = React.forwardRe
           onFocus={onSelectStar}
           onClick={onSelectStar} // For Safari & Firefox on OSX
           disabled={!!(disabled || readOnly)}
-          role="presentation"
+          role="radio"
+          aria-hidden={readOnly ? 'true' : undefined}
           type="button"
+          aria-checked={starNum === Math.ceil(displayRating) ? true : false}
```

Comment:
nit, it's generally preferred in cases like this to just use the result of the equality check directly
```suggestion
          aria-checked={starNum === Math.ceil(displayRating)}
```